### PR TITLE
Add theme.js file and ThemeProvider

### PR DESCRIPTION
**Pull request by martinyis**
I've made some updates to the project to improve the overall look and feel. Specifically, I've added a theme.js file that includes color and font information for the project, and I've imported it into App.js and wrapped the App component with ThemeProvider.

In addition, I've made some updates to several components to ensure that the Roboto font is used consistently throughout the project.

Please let me know if there's anything else you'd like me to add or change.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
@martinyis not getting website hero section in staging build of this PR would you look into this why it is not showing also those font are default which we don't follow we use `Google Sans` or `Open Sans` only


https://user-images.githubusercontent.com/71271069/226066558-721ce028-8f25-4ce6-9a04-5bc019bd5e5b.mov

File: src/theme.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        600: "#666666",
+        700: "#525252",
+        800: "#333333",
+      },
+    },
+    eventBackground: "#F6F6F6",
+    teamsBackground: "#EAEAEA",
+  },
+  typography: {
+    fontFamily: "Roboto",
```
Comment (Yashsharma1911) [inline src/theme.js:36]: here we need Google sans

**Comment by Yashsharma1911** [comment]
By the way, this looks nice having theme you have done some good work here 👏

**Comment by Yashsharma1911** [comment]
This PR fixes #69

**Comment by martinyis** [comment]
@Yashsharma1911 I will take a look at it and will try to fix it.

**Pull request closed without merging**